webpay/base/templates/base.html

```diff
@@ -84,10 +84,5 @@ <h2 class="heading"></h2>
       <div class="throbber"></div>
       <span class="txt"></span>
     </div>
-
-    {% block site_js %}
-      <script src="{{ settings.BROWSERID_JS_URL }}"></script>
-      {{ js('pay') }}
-    {% endblock %}
```

Comment:
Can we just delete this whole template and anything that loads a media file? Or is that a rabbit hole for another bug?